internal/cmd/gocdk/init_test.go

```diff
@@ -198,8 +198,9 @@ func TestInferModulePath(t *testing.T) {
 					t.Error(err)
 				}
 			}()
```

Comment:
General comment: it would be nice to test for both the "src" and non-"src" variant as a regression test.